src/themes/base/components/Text/textVariables.ts

```diff
@@ -47,8 +51,11 @@ export default (siteVariables): TextVariables => {
     errorColor: siteVariables.colors.red[500],
     successColor: siteVariables.colors.green[500],
 
-    fontSizeExtraSmall: siteVariables.fontSizes.smaller,
-    fontLineHeightExtraSmall: siteVariables.lineHeightExtraSmall,
+    fontSizeSmallest: siteVariables.fontSizes.smallest,
+    fontLineHeightSmallest: siteVariables.lineHeightExtraSmall,
+
+    fontSizeSmaller: siteVariables.fontSizes.smaller,
+    fontLineHeightSmaller: siteVariables.lineHeightSmaller,
```

Comment:
just to be sure - thus, speaking of the `lineHeight` range, we will have values from `smaller` to `small`, right?